aviary/variable_info/variable_meta_data.py

```diff
@@ -2116,12 +2127,13 @@
 add_meta_data(
     Aircraft.Engine.PROPELLER_TIP_MACH_MAX,
     meta_data=_MetaData,
-    historical_name={"GASP": None,  # TODO this needs verification
-                     "FLOPS": None,
-                     "LEAPS1": None
-                     },
+    historical_name={
+        "GASP": None,  # TODO this needs verification
+        "FLOPS": None,
+        "LEAPS1": None,
+    },
     units='unitless',
-    desc='maximum allowable Mach number at propeller tip (based on helical speed)',
+    desc='maximum allowable Mach number at propeller tip (based on helical airspeed)',
```

Comment:
I believe this should just be "speed", because the helical speed is a combination of the airspeed and the rotational speed

Reply:
Will update for clarity